### ISSUE TITLE
GHC 9.2 support?

### ISSUE THREAD

**Issue by kozross** [well-typed/cborg#272]
Currently, `binary-serialise-cbor` won't build with GHC 9.2.1:

```
Resolving dependencies...
cabal: Could not resolve dependencies:
[__0] trying: binary-serialise-cbor-0.2.1.0 (user goal)
[__1] next goal: base (dependency of binary-serialise-cbor)
[__1] rejecting: base-4.16.0.0/installed-4.16.0.0 (conflict:
binary-serialise-cbor => base>=4.8 && <4.16)
[__1] skipping: base-4.16.0.0 (has the same characteristics that caused the
previous version to fail: excluded by constraint '>=4.8 && <4.16' from
'binary-serialise-cbor')
[__1] rejecting: base-4.15.0.0, base-4.14.3.0, base-4.14.2.0, base-4.14.1.0,
base-4.14.0.0, base-4.13.0.0, base-4.12.0.0, base-4.11.1.0, base-4.11.0.0,
base-4.10.1.0, base-4.10.0.0, base-4.9.1.0, base-4.9.0.0, base-4.8.2.0,
base-4.8.1.0, base-4.8.0.0, base-4.7.0.2, base-4.7.0.1, base-4.7.0.0,
base-4.6.0.1, base-4.6.0.0, base-4.5.1.0, base-4.5.0.0, base-4.4.1.0,
base-4.4.0.0, base-4.3.1.0, base-4.3.0.0, base-4.2.0.2, base-4.2.0.1,
base-4.2.0.0, base-4.1.0.0, base-4.0.0.0, base-3.0.3.2, base-3.0.3.1
(constraint from non-upgradeable package requires installed instance)
[__1] fail (backjumping, conflict set: base, binary-serialise-cbor)
After searching the rest of the dependency tree exhaustively, these were the
goals I've had most trouble fulfilling: base, binary-serialise-cbor
```

**Comment by bgamari** [comment]
I'll try to get this sorted today.

**Comment by bgamari** [comment]
Handled in #273.

**Comment by bgamari** [comment]
Released the following packages with GHC 9.2 support:

 * `serialise-0.2.4.0`
 * `cborg-0.2.6.0`
 * `cborg-json-0.2.3.0`
